Fix Firefox UI issues

denisw opened the pull request:
This branch contains two commits which fix UI issues which show up in Firefox and other Gecko-based browsers.

Commit:
Do not rely on "position: relative" td for channel title

According to the CSS spec, the effect of "position: relative"
on a td element is undefined. Firefox seems to ignore it, which
means that the channel title positioning used in the channel title
("position: absolute" div inside of "position: relative" td) doesn't
work here. Using just normal positioning is safer and changes the
layout only by a few pixels, so use that.

Commit:
Use "overflow: auto" instead of "overflow: scroll" for channel list

Using "overflow: scroll" here always shows both a vertical and
horizontal scrollbar in the channel list even if there is only a
single channel there.

dodo commented on src/styles/antiscroll.styl:
```diff
@@ -32,7 +32,7 @@
 
 .antiscroll-inner {
-  overflow: scroll;
+  overflow: auto;
```
that's wrong, antiscroll requires `overflow: scroll`  ( https://github.com/LearnBoost/antiscroll )

denisw replied:
I thought native scrollbars were the intended behavior, but I understand now. This will be reverted.

dodo commented on src/styles/stream.styl:
```diff
@@ -42,12 +42,6 @@ body > .content
-        width 100%
-        left 0
-        top 0
-        position absolute
```
these changes brake the title bar behavior for channels without a status node, like https://beta.buddycloud.org/private@buddycloud.org

the channel title should be always at the same position.

denisw replied:
Oh, I didn't know that this case is possible. I'll try to find a way to fix this.

denisw replied:
How about giving the .titleText td "vertical-align: top"? The title position would be fixed in this case.

Commit:
Revert "Use "overflow: auto" instead of "overflow: scroll" for channel list"

I didn't recognize that this is part of Antiscroll, which just doesn't
work as expected in the moment.

This reverts commit d1d7d0dd8863d7e8c759c3df1d2c997869c9e973.

imaginator commented:
Thanks Denis! This is great.

Commit:
Use "vertical-align: top" for the channel title

This ensures that the title is always at the same position,
even if there is no channel status text.

Commit:
Make sure the "Loading..." message is centerd in Firefox

Due to differences in the Firefox flexbox implementation, the
loading message was stuck in the upper left corner.

Commit:
Merge remote-tracking branch 'upstream/master'
